### PR TITLE
fix(defender service): retrieving key dicts with get

### PR DESCRIPTION
**Pull request by n4ch04**
### Context

It seems to be some problems with certain Azure defender service configuration, some dict keys were not present

### Description

Fixes `defender` checks including `get` method for dict keys

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/providers/azure/services/defender/defender_ensure_defender_for_databases_is_on/defender_ensure_defender_for_databases_is_on.py [hunk 1]
```diff
@@ -6,19 +6,26 @@ class defender_ensure_defender_for_databases_is_on(Check):
...
-            report.status_extended = f"Defender plan Defender for Databases from subscription {subscription} is set to ON (pricing tier standard)"
             if (
-                pricings["SqlServers"].pricing_tier != "Standard"
-                or pricings["SqlServerVirtualMachines"].pricing_tier != "Standard"
-                or pricings["OpenSourceRelationalDatabases"].pricing_tier != "Standard"
-                or pricings["CosmosDbs"].pricing_tier != "Standard"
+                "SqlServers" in pricings
+                and "SqlServerVirtualMachines" in pricings
+                and "OpenSourceRelationalDatabases" in pricings
+                and "CosmosDbs" in pricings
```
Comment (MrCloudSec) [inline prowler/providers/azure/services/defender/defender_ensure_defender_for_databases_is_on/defender_ensure_defender_for_databases_is_on.py:13]: I would check one by one.

**Review by MrCloudSec — CHANGES_REQUESTED** [review]

File: prowler/providers/azure/services/defender/defender_ensure_defender_for_databases_is_on/defender_ensure_defender_for_databases_is_on.py (hunk 1 above)
Comment (n4ch04) [reply prowler/providers/azure/services/defender/defender_ensure_defender_for_databases_is_on/defender_ensure_defender_for_databases_is_on.py:13]: It needs the four pricings to be enabled, so there is no point in checking it one by one

**Review by MrCloudSec — DISMISSED** [review]

**Review by MrCloudSec — DISMISSED** [review]

**Review by MrCloudSec — DISMISSED** [review]
🔝 🔝 🔝

**Review by MrCloudSec — APPROVED** [review]
🔝 🔝

**Pull request merged**